docs/review.md

```diff
@@ -0,0 +1,67 @@
+# Review Guidelines
+
+The following summarises review guidelines that we follow for pull requests in
+Riot Web and other supporting repos. These are just guidelines (not strict
+rules) and may be updated over time.
+
+## Code Review
+
+When reviewing code, here are some things we look for and also things we avoid:
+
+### We review for
+
+* Correctness
+* Performance
+* Accessibility
+* Security
+* Comments and documentation where needed
+* Sharing knowledge of different areas among the team
+* Ensuring it's something we're comfortable maintaining for the long term
+
+### We should avoid
+
+* Style nits that are already handled by the linter
+* Dramatically increasing scope
+
+### Good practices
+
+* Use empathetic language
+  * See also [Mindful Communication in Code
+    Reviews](https://kickstarter.engineering/a-guide-to-mindful-communication-in-code-reviews-48aab5282e5e)
+    and [How to Do Code Reviews Like a Human](https://mtlynch.io/human-code-reviews-1/)
+* Authors should prefer smaller commits for easier reviewing and bisection
+* Reviewers should be explicit about required versus optional changes
+  * Reviews are conversations and the PR author should feel comfortable
+    discussing and pushing back on changes before making them
+* Core team should lead by example through their tone and language
+* Take the time to thank and point out good code changes
+* Using softer language like "please" and "what do you think?" goes a long way
+  towards making others feel like colleagues working towards a common goal
+
+### Workflow
+
+* Avoid force pushing to a PR after first round of review
+* Use merge commits when landing
+* PR author merges after review (assuming they have write access)
+* Assign issues only when in progress (don’t overly assign things so it’s clear
+  that anyone on the team can pick up)
+
+## Design and Product Review
```

Comment:
Can we stick something in here for community members to ask before they embark on a feature? This might need to go into the contributing guidelines as well/instead.

We have a large number of open issues these days, and open issues don't always mean we're okay with the suggestion/feature request/idea from 5 years ago. A quick check in #riot-dev where us devs can bring in the product team would be greatly appreciated.

Reply:
Yes, that's a great point. I added a note here and also on the README... Not quite sure where people are most likely to see it, so may need some tuning over time.